README.md

```diff
@@ -83,6 +83,26 @@ The following changes are now available in production:
 
 ### individuals-charges-api
 
+New Endpoint: `Delete Pension Charges` - Tax Year Specific 
+
+Updated Endpoint: `Amend Pension Charges`, see [sandbox changelog](https://github.com/hmrc/income-tax-mtd-changelog/wiki#22nd-february-2023).
+
+Updated Endpoint: `Retrieve Pension Charges`, see [sandbox changelog](https://github.com/hmrc/income-tax-mtd-changelog/wiki#22nd-february-2023).
+
+Minor clarifications made to developer hub documentation:
+
+* Updated description of Delete Pension Charges endpoint.
+
+Minor clarifications made to developer hub documentation:
```

Comment:
This developer hub update can be merged with the one above (i.e. One minor clarifications title with two bullet points).